storage/store.go

```diff
@@ -1783,6 +1783,20 @@ func (s *Store) computeReplicationStatus(now int64) (
 	}
 
 	timestamp := roachpb.Timestamp{WallTime: now}
+	raftStatusMap := make(map[roachpb.RangeID]*raft.Status)
+	s.mu.Lock()
+	for rangeID := range s.replicas {
+		raftStatusMap[rangeID] = &raft.Status{}
```

Comment:
can you assign nil here? seems like that would save some allocations and would be otherwise identical